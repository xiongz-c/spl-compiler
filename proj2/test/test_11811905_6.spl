struct test {
    int m1;
};

int func(struct test s1, int s2) {
    return s1.m1 + s2;
}

int func() {}

int test_11811806_6(int v1) {
    struct test2 ttt;
    struct test v2;
    v2.m1 = v1;

    v2+1 = 2.0;
    return func(v2, 1.0);
}