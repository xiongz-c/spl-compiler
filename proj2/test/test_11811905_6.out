Error type 4 at Line 9: redefine function: func
Error type 14 at Line 12: no such member
Error type 7 at Line 16: binary operation on non-number variables
Error type 6 at Line 16: left side in assignment is rvalue
Error type 5 at Line 16: unmatching type on both sides of assignment
Error type 9 at Line 17: invalid argument type for compare in function func
